### ISSUE TITLE
Dynamically changing items 'static' property

### ISSUE THREAD

**Issue by MintyMods** [vaheqelyan/svelte-grid#10]
Hi Vahe,

I have a requirement to make an item static based on the users interaction. Basically allow them to 'pin' an item into place. The use case is that they would add a chart to my page then position it within the other charts before pinning it into place where it would remain as a static item. 

I have tried changing the `static` property using the following code with no luck:-

```
 function pinItemContainer(item) {
    item.static = !item.static; 
    gridHelp.resizeItems(items, cols);   
  }
```

Is dynamically changing the static attribute possible?

**Comment by vaheqelyan** [comment]
[Repl](https://svelte.dev/repl/c993f216cbe047b3934646c67770bae4?version=3.12.1).
But I don't know why svelte doesn't remove event listeners and react correctly
By the way `gridHelp.resizeItems` returns an array of widgets that it does not mutate.

Everything seems to be done correctly.

https://github.com/vaheqelyan/svelte-grid/blob/3a7404bd0808fdc86ed44fa16b47c15ae59ceb81/src/index.svelte#L49

https://github.com/vaheqelyan/svelte-grid/blob/3a7404bd0808fdc86ed44fa16b47c15ae59ceb81/src/index.svelte#L60#61
https://github.com/vaheqelyan/svelte-grid/blob/3a7404bd0808fdc86ed44fa16b47c15ae59ceb81/src/index.svelte#L60

**Comment by vaheqelyan** [comment]
This is caused by mouseup.

**Comment by vaheqelyan** [comment]
This is an updated version [repl](https://svelte.dev/repl/c993f216cbe047b3934646c67770bae4?version=3.12.1)

**Comment by vaheqelyan** [comment]
Really don't know why svelte doesn't respond to event listeners

**Comment by vaheqelyan** [comment]
well, i found out that you can use [tick](https://svelte.dev/docs#tick), and it will solve your problem. but if you want to make a static element not a static element it will not work.
I think this is due to svelte. [repl](https://svelte.dev/repl/b3f0972b61a048169a684c0f6277b720?version=3.12.1), or am I doing wrong in my component by attaching event listeners as a condition

**Comment by MintyMods** [comment]
I do require a toggle of the static state so will watch the issue you have raised.
Thanks

**Comment by vaheqelyan** [comment]
I have update the svelte to the latest version. They seem to fix the problem. Try it if everything is OK we can close this issue.

**Comment by MintyMods** [comment]
Hi @vaheqelyan

Upgrading to the latest release of Svelte `3.16.0` does indeed resolves this issue with the static property not being dynamically applied. 

I notice you have just released `1.10.6` so I will install that shortly to get fixes #17, #4 and retest my remaining issues _(I closed #8 as was unable to reproduce using a different router)_.

Thank you for your help with this.
